homeassistant/components/google_assistant/__init__.py

```diff
@@ -48,6 +49,7 @@
         vol.Optional(CONF_EXPOSE, default=True): cv.boolean,
         vol.Optional(CONF_ALIASES): vol.All(cv.ensure_list, [cv.string]),
         vol.Optional(CONF_ROOM_HINT): cv.string,
+        vol.Optional(CONF_IGNORE_SECURE_DEVICE_PIN, default=False): cv.boolean,
```

Comment:
Instead of making this an ignore pin option, make it a secure device override option, allowing both true or false

Reply:
This is definetly something i can do, but it requires far more substantial code changes and also tests (which i guess i can figure out, but i don't have experience with them so i'm starting from scratch there) so i'm not so sure when i will have the time for this